.circleci/config.yml

```diff
@@ -0,0 +1,54 @@
+version: 2
+
+jobs:
+  test:
+    docker:
+      - image: artsy/hokusai:0.4.0
```

Comment:
you can go up to 0.4.3 which keeps the container around after test so you can get artifacts if needed.